shared/xlink/xlink_wrapper.cpp

```diff
@@ -39,6 +39,13 @@ XLinkWrapper::~XLinkWrapper()
             XLinkWrapper::rebootDevice(_device_link_id);
         }
 #endif
+    
+        XLinkError_t err = XLinkResetRemote(_device_link_id);
```

Comment:
Guard the function call by the above `#ifdef __PC__`, to not break the build on device.

Reply:
Actually `XLinkWrapper::rebootDevice` from above seems to call the same function `XLinkResetRemote`, if enabled by `_reboot_device_on_destructor` (from py_bindings.cpp). Should we enable it there instead?

Reply:
yes